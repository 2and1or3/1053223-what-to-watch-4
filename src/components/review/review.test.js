import React from "react";
import renderer from "react-test-renderer";

import Review from './review.jsx';

const comment = {
  id: `0`,
  author: `Kate Muir`,
  date: `Month dd, yyyy`,
  description: `The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.`,
  rate: `8,9`,
};

it(`Render Review component`, () => {
  const tree = renderer
    .create(<Review comment = {comment}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

// npm run test.jest -- components/review/review.test.js
