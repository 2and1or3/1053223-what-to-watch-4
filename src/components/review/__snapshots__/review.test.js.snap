// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Review component 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      The mannered, madcap proceedings are often delightful, occasionally silly, and here and there, gruesome and/or heartbreaking.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Muir
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-24"
      >
        Month dd, yyyy
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,9
  </div>
</div>
`;
